src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private final FruitDao fruitDao = new FruitDaoImpl();
+    private final Map<Fruit, Integer> fruitsStorage = Storage.fruitsStorage;
```

Comment:
I am not sure, but i think you can contact to Storage.fruitsStorage directly or if you want local variable : create  deep copy of storage.